.github/actions/poll-for-status/action.yaml

```diff
@@ -0,0 +1,58 @@
+name: Poll for status of workflow
+description: This action will wait and poll for the status of workflow
```

Comment:
Can you fill me in on what the purpose of this is, or why it's needed? Is this for checking the status of a remote workflow in another repo? Also, are there potential issues with this approach if the scope of the PAT changes?

Reply:
This code is no longer being used due to the built in github keyword `needs` being used in its placed. Removed action from code.